sync/class.jetpack-sync-defaults.php

```diff
@@ -81,7 +81,15 @@ class Jetpack_Sync_Defaults {
 		'disabled_reblogs',
 		'jetpack_comment_likes_enabled',
 		'twitter_via',
-		'twitter-cards-site-tag'
+		'twitter-cards-site-tag',
+		'wpcom_publish_posts_with_markdown',
```

Comment:
Most of these new options don't seem to have associated tests. That's fine I guess, since most of them didn't have tests in the first place, but it's the kind of thing I'd like to avoid in future.